pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -123,6 +129,12 @@ func (o *OAuthAPIServerOptions) NewOAuthAPIServerConfig() (*apiserver.Config, er
 
 	serverConfig := apiserver.NewConfig()
 	serverConfig.GenericConfig.OpenAPIConfig = openapiconfig.DefaultOpenAPIConfig()
+	serverConfig.GenericConfig.OpenAPIV3Config = openapiconfig.DefaultOpenAPIConfig()
+	serverConfig.GenericConfig.AggregatedDiscoveryGroupManager = aggregated.NewResourceManager("apis")
```

Comment:
shouldn't this be featuregated?

Reply:
As far as I can see, this was also discussed in openshift-apiserver bump and concluded that featuregated check was done in generic-apiserver https://github.com/openshift/openshift-apiserver/pull/371#discussion_r1228083743

Reply:
now it seems to me that we don't even need it, because it is initialized later 

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/config.go#L746

will investigate

we also need to update the openshift-apiserver: https://github.com/openshift/openshift-apiserver/pull/371/commits/b746cade130cd6f3a8b33c0700613bb7b83a9a8d#diff-d9072b7a1f5c28fd0867788847343611dbdd5b5a30f2fcd858cedb6655776fb9R108

But we probably do not need feature gate for the openapi as it is already GA and locked to True, right?

Reply:
I just read your comment @ardaguclu. Seems this was resolved in the openshift-apiserver discussion